.github/workflows/python-package.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     env:
       DJANGO_SETTINGS_MODULE: pinakes.settings.development
       PINAKES_SECRET_KEY: test
+      # Base64 encoded b'TEST' * 8
```

Comment:
Is this a valid key generated by `Fernet.generate_key()`?

Reply:
Absolutely.

Reply:
@cutwater Can you add this key (or the path for this file) to the `.gitleaks.toml` file to indicate that it is allowed? Otherwise it might get flagged by infosec/prodsec as a false positive.